fineract-provider/src/main/java/org/apache/fineract/cob/service/InlineLoanCOBExecutorServiceImpl.java

```diff
@@ -66,7 +66,7 @@
 @Service
 @Slf4j
 @RequiredArgsConstructor
-public class InlineLoanCOBExecutorServiceImpl implements InlineExecutorService, InitializingBean {
+public class InlineLoanCOBExecutorServiceImpl implements InlineExecutorService<List<Long>>, InitializingBean {
```

Comment:
What if we  simply don't specify the List part of the generic parameter? Just the Long. That way all the interface implementors will have the same List input type for the method.